docs/getting_started/limitations.md

```diff
@@ -16,9 +16,15 @@ The Cloud Foundry command line client has an option to enable you to connect dir
 
 This functionality is currently disabled on the Government PaaS. We intend to make it available in future, following security testing.
 
-## 404s after push or restart
+## 404s after commands that restart the app
 
-After you use ``cf push`` or ``cf restart``, your app may briefly return incorrect 404 errors. Apart from the brief downtime, this may lead to problems if the 404 is cached, or visiting web crawling bots (as used by search engines) receive a 404.
+After you use a command that restarts application instances, such as ``cf push`` or ``cf restart``, your app may briefly return incorrect 404 errors. Apart from the brief downtime, this may lead to problems if the 404 is cached, or visiting web crawling bots (as used by search engines) receive a 404.
```

Comment:
technically it's not 'your app' returning the 404s. I'd say, you might get 404s when trying to access your app